src/cryptography/encrypt.js

```diff
@@ -133,10 +133,33 @@ const getTagBuffer = tag => {
 	return tagBuffer;
 };
 
-const decryptAES256GCMWithPassword = (
-	{ iterations = PBKDF2_ITERATIONS, cipherText, iv, salt, tag },
-	password,
-) => {
+const validateEncryptedPassphrase = ({ cipherText, iv, salt, tag }) => {
+	const requiredProperties = [
+		[cipherText, 'Cipher text'],
+		[iv, 'IV'],
+		[salt, 'Salt'],
+		[tag, 'Tag'],
+	];
+
+	requiredProperties.forEach(([property, propertyName]) => {
+		try {
+			hexToBuffer(property);
```

Comment:
Wouldn't it weird to have partial validation named `validateEncryptedPassphrase`.
For example, `getTagBuffer` still validates if the length is 16, which is still used in encrypted passphrase, but not validated here.

Reply:
Yes. How would you suggest doing it instead?

Reply:
oops sorry, forgot to put the suggestion.
Since we have method for each one with separate logic, maybe we should put the validation logic with meaningful message on each method?